pkgs/tools/misc/pc/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, byacc
+, fetchFromSourcehut
+, gitUpdater
+}:
+
+stdenv.mkDerivation {
+  pname = "pc";
+  version = "0.4";
+
+  src = fetchFromSourcehut {
+    owner = "~ft";
+    repo = "pc";
+    rev = "0.4";
```

```suggestion
    rev = finalAttrs.version;
```
Please use a variable here since we'll forget to update it.